src/template.js

```diff
@@ -562,7 +583,18 @@ module.exports = {
             'p-mainapplicant-authority': mainApplicantAuthorityToApply.route,
             'p--download-your-answers': downloadAnswers.route,
             'p--has-legal-authority': flowHasLegalAuthority.route,
-            'p--represents-legal-authority': flowRepresentsLegalAuthority.route
+            'p--represents-legal-authority': flowRepresentsLegalAuthority.route,
+            'p--context-deceased-details': contextDeceasedDetails.route,
+            'p-deceased-name': deceasedName.route,
+            'p-deceased-date-of-birth': deceasedDateOfBirth.route,
+            'p-deceased-date-of-death': deceasedDateOfDeath.route,
+            'p-deceased-address': deceasedAddress.route,
+            'p-context-funeral-costs': contextFuneralCost.route,
```

Comment:
Context pages are generally of the form p--context rather than p-context.
I realise that the data dictionary has this defined as p-context-funeral-costs perhaps this is an oversight or intent?